ilan/build

```diff
@@ -21,15 +21,13 @@ if [ $# != 3 ]; then
   exit 1
 fi
 
-docker build -t installation-manager im || exit $?
-docker run --rm -v $(pwd):/host installation-manager /host/install_ihs $1 $2 $3 || exit $?
+docker build -t installation-manager . --build-arg VERSION=$1 --build-arg IBM_ID=$2  --build-arg IBM_PASSWORD=$3|| exit $?
 
-docker run -d --name tar_server -v $(pwd)/ihs$1.tar.gz:/host/ihs$1.tar.gz -w /host python:2-slim python -m SimpleHTTPServer
-tar_server_ip=$(docker inspect --format '{{ .NetworkSettings.IPAddress }}' tar_server)
-tar_url="http://${tar_server_ip}:8000/ihs${1}.tar.gz"
+# copy the intermediate artifacts from /target
 
+docker run installation-manager cat /target/ihs$1.tar.gz > im/ihs$1.tar.gz
+docker run installation-manager cat /target/Dockerfile.$1 > im/Dockerfile.$1
 
 # Build image from hosted tar file
 echo "Building image"
-docker build -t ibm-http-server:$1 --build-arg TAR_URL=$tar_url . || exit $?
-docker rm -f tar_server
+docker build -t ibm-http-server:$1 -f im/DockerFile.$1 . || exit $?
```

Comment:
I could add `--squash` here to get rid of the intermediate images but the feature is still experimental in Docker 1.13 so I opted not to for now.  The end result for the end users will not have the ihs.tar.gz file regardless.